pod/video/templates/videos/video-info.html

```diff
@@ -242,6 +242,16 @@ <h2 class="tab-title"><i data-feather="share-2"></i>&nbsp;{% if video.is_draft %
             </div>
             <small id="displaytimeHelp" class="form-text text-muted">{% trans 'Check the box to indicate the beginning of playing desired.' %}</small>
           </div>
+	  <legend><i data-feather="upload-cloud""></i>&nbsp;{% trans 'BigBlueButton/H5P Integration' %}</legend>
```

Comment:
Sauf erreur de ma part, il manque juste avant cette ligne : 
{% if request.user ==  video.owner or request.user in video.additional_owners.all%}
et à la fin 
{% endif %}

Reply:
ces elements sont es accès idrect au fichier encodés et ne doivent etre donc montré qu'aux propriétaires qui souhaitent utiliser ces fichiers dans le cadre BBB ou H5P par exemple

Reply:
@jaworskyp des news ?